nifty-core/src/main/java/com/facebook/nifty/ssl/SSLClientConfiguration.java

```diff
@@ -0,0 +1,89 @@
+package com.facebook.nifty.ssl;
+
+import com.google.common.base.Throwables;
+import org.jboss.netty.handler.ssl.SslContext;
+import org.jboss.netty.handler.ssl.SslHandler;
+
+import javax.net.ssl.SSLException;
+import java.io.File;
+import java.net.InetSocketAddress;
+import java.net.SocketAddress;
+
+public class SSLClientConfiguration {
```

Comment:
Like URI or like Url? ☺

Javax.net.ssl.SSLException?

From: Dain Sundstrom <notifications@github.com>
Reply-To: facebook/nifty <reply@reply.github.com>
Date: Tuesday, June 14, 2016 at 8:28 PM
To: facebook/nifty <nifty@noreply.github.com>
Cc: Andrew Cox <andrewcox@fb.com>, Mention <mention@noreply.github.com>
Subject: Re: [facebook/nifty] Add ssl support to nifty (#154)


In nifty-core/src/main/java/com/facebook/nifty/ssl/SSLClientConfiguration.java<https://github.com/facebook/nifty/pull/154#discussion_r67094745>:

> @@ -0,0 +1,89 @@

> +package com.facebook.nifty.ssl;

> +

> +import com.google.common.base.Throwables;

> +import org.jboss.netty.handler.ssl.SslContext;

> +import org.jboss.netty.handler.ssl.SslHandler;

> +

> +import javax.net.ssl.SSLException;

> +import java.io.File;

> +import java.net.InetSocketAddress;

> +import java.net.SocketAddress;

> +

> +public class SSLClientConfiguration {

SSL is a abbreviation and should be capitalized as a single word Ssl.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/facebook/nifty/pull/154/files/d7d4417da0574fb08cfbaeb5addc9b97307d7d85#r67094745>, or mute the thread<https://github.com/notifications/unsubscribe/ABo_Hk0CEQJqBQAUvT5Q9-lEfxpqQzVjks5qL3FMgaJpZM4IzLEa>.